specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json

```diff
@@ -507,6 +507,53 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/quotaType/{quotaType}/usages": {
+      "get": {
+        "summary": "Gets subscription core usages for location, for a given type of quota.",
+        "description": "Gets the usage in cores for a subscription in a given location, for a specific quota type.",
+        "operationId": "GetUsagesInLocation_list",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/LocationParameter"
+          },
+          {
+            "name": "quotaType",
+            "description": "The type of quota to get usages for",
```

Comment:
What does a quota type string look like? How do people know they're using the right one?

Reply:
For now quota type is "regional" or "zonal". We may add support for other types if our quota construct is expanded. We will return an appropriate error message for invalid quota type string.

Reply:
I think you should define this as an enum so that its obvious to the API users what values to include.